pkgs/by-name/li/lib25519/package.nix

```diff
@@ -31,7 +37,10 @@ stdenv.mkDerivation (finalAttrs: {
     runHook postConfigure
   '';
 
-  nativeBuildInputs = [ python3 ];
+  nativeBuildInputs = [
+    python3
+    valgrind
```

Comment:
Why is `valgrind` necessary?

Reply:
It's a build dependency of the testing component of the project.

Reply:
Thanks for the explanation.